### DISCUSSION TITLE
CSS를 적용하는 방법에 대해 설명해주세요.

### DISCUSSION THREAD

**Discussion by BaekYuri** [wanted-9th-3team/frontend-study#49] [프론트 기술 질문 공간]
2가지 이상 설명해주세요.

**Comment by michoball** [comment]
1. 따로 css 파일을 만들어서 스타일링을 하고 index.html 폴더에 link 태그로 적용하는 방법이 있습니다. 

2. HTML 태그 내부에 style 요소로 스타일을 적용할 수 있습니다. 

3. HTML 파일 head에 style 태그 내에 스타일링을 할 수 있습니다.

**Comment by WooGie911** [comment]
세가지가 있습니다.

1. Inline Style Sheet
 : HTML 태그의 style 속성에 CSS 코드를 넣는 방법입니다.
`<p style="color: blue">p tag</p>`

2. Internal Style Sheet
 : HTML 문서 안의 <style>과 </style> 안에 CSS 코드를 넣는 방법입니다.
```
<style>
  h1 {
    color: blue;
  }
</style>
```

3. Linking Style Sheet
별도의 CSS 파일을 만들고 HTML 문서와 연결하는 방법입니다.

**Comment by BaekYuri** [comment]
1. html 태그 안에 styles 속성을 이용하기
2. html 파일 내에 <style> 태그 사용하기
3. .css 파일 생성하고 따로 link를 이용하여 적용하기
